pages/directory/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { Box, Container, List, ListItem, Typography } from '@mui/material';
+import { NextPage } from 'next';
+import React from 'react';
+
+import createAxiosInstance from 'client/lib/axios';
+
+//TODO: Type users once sequelize-typescript is added
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const Directory: NextPage<{ users: any[] }> = props => {
```

Comment:
Lint did not like users: any. 

I added an ignore as this should be a temporary bandaid until sequilize-typscript